docsteady/ve_baseline.py

```diff
@@ -138,10 +138,9 @@ def get_ve_details(rs, key):
 
     # cache reqs
     if 'req_id' in ve_details.keys():
-        ve_long_name = ve_details['summary'].split(":")
         if ve_details["req_id"] not in Config.CACHED_REQS_FOR_VES:
             Config.CACHED_REQS_FOR_VES[ve_details["req_id"]] = []
-        Config.CACHED_REQS_FOR_VES[ve_details["req_id"]].append(ve_long_name[0])
+        Config.CACHED_REQS_FOR_VES[ve_details["req_id"]].append(ve_details["key"])
```

Comment:
Took me a long time to find where this is used: https://github.com/lsst-dm/docsteady/blob/main/docsteady/vcd.py#L602

Reply:
my issue was in the jinja template for ve-baseline.